activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -286,7 +286,8 @@ public class BrokerService implements Service {
                 LOG.info("Loaded the Bouncy Castle security provider at position: {}", ret);
             }
         } catch(Throwable e) {
-            // No BouncyCastle found so we use the default Java Security Provider
+            // No BouncyCastle found, so we use the default Java Security Provider
+            LOG.info("Using the default Java security provider");
```

Comment:
Does this need to be printed out on every startup?  Perhaps this is debug level?

Reply:
Good point. Changed it to debug.